patches/net/minecraft/world/entity/SpawnPlacements.java.patch

```diff
@@ -18,18 +21,16 @@
      }
  
      static {
-@@ -168,5 +_,14 @@
+@@ -168,5 +_,12 @@
      @FunctionalInterface
      public interface SpawnPredicate<T extends Entity> {
          boolean test(EntityType<T> p_217081_, ServerLevelAccessor p_217082_, MobSpawnType p_217083_, BlockPos p_217084_, RandomSource p_217085_);
 +    }
 +
-+    // ******* FORGE START. INTERNAL USE ONLY! ****** //
```

Comment:
If you're going to delete the "internal use only" comment, this at least needs an `@ApiStatus.Internal` marker or similar.